src/checkfiles/checkfiles.py

```diff
@@ -184,11 +184,16 @@ def check_md5sum(expected_md5sum, calculated_md5sum):
     return error
 
 
+def make_content_md5sum_search_url(content_md5sum, uuid, portal_url):
+    search_url = portal_url + \
```

Comment:
Why not move protal_url into format string as well?

Reply:
moved